# A representation of first-order variations in the vehicle's position
# (usually referred to as surge, sway, and heave), orientation, and in
# both of their rates of change.

auv_interfaces/GeneralizedVectorWithCovariance displacement
auv_interfaces/GeneralizedVectorWithCovariance rate
